doc/index.rst

```diff
@@ -62,3 +62,12 @@ Amazon provides implementations of some common machine learning algortithms opti
     factorization_machines
     lda
     ntm
+
+SageMaker Workflow
```

Comment:
would it make more sense to change this to just "Workflows"? (the workflow tools themselves aren't part of SageMaker)

Reply:
Yep, this is better. 

Updated! Thanks!